### ISSUE TITLE
object listing has incorrect format for last_modified

### ISSUE THREAD

**Issue by andrewgaul** [bouncestorage/swiftproxy#33]
swiftproxy returns seconds since epoch `1433809946492` instead of ISO 8601 `2014-01-15T16:41:49.390270`.

**Comment by andrewgaul** [comment]
This causes cascading failures in jclouds integration tests during clear container:

```
Caused by: java.text.ParseException: Unparseable date: "1433809416447+0000"
        at java.text.DateFormat.parse(DateFormat.java:366)
        at org.jclouds.date.internal.SimpleDateFormatDateService.iso8601SecondsDateParse(SimpleDateFormatDateService.java:164)
        at org.jclouds.json.config.GsonModule$Iso8601DateAdapter.parseDate(GsonModule.java:241)
        at org.jclouds.json.config.GsonModule$Iso8601DateAdapter.read(GsonModule.java:234)
        at org.jclouds.json.config.GsonModule$Iso8601DateAdapter.read(GsonModule.java:220)
        at com.google.gson.TypeAdapter$1.read(TypeAdapter.java:199)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:95)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:183)
        at org.jclouds.json.internal.NullFilteringTypeAdapterFactories$IterableTypeAdapter.readAndBuild(NullFilteringTypeAdapterFactories.java:90)
        at org.jclouds.json.internal.NullFilteringTypeAdapterFactories$IterableTypeAdapter.read(NullFilteringTypeAdapterFactories.java:82)
        at org.jclouds.json.internal.NullFilteringTypeAdapterFactories$IterableTypeAdapter.read(NullFilteringTypeAdapterFactories.java:61)
        at com.google.gson.Gson.fromJson(Gson.java:805)
        at com.google.gson.Gson.fromJson(Gson.java:770)
        at com.google.gson.Gson.fromJson(Gson.java:719)
        at org.jclouds.json.internal.GsonWrapper.fromJson(GsonWrapper.java:42)
        at org.jclouds.http.functions.ParseJson.apply(ParseJson.java:83)
        at org.jclouds.http.functions.ParseJson.apply(ParseJson.java:77)
        at org.jclouds.http.functions.ParseJson.apply(ParseJson.java:62)
        at org.jclouds.openstack.swift.v1.functions.ParseObjectListFromResponse.apply(ParseObjectListFromResponse.java:68)
        at org.jclouds.openstack.swift.v1.functions.ParseObjectListFromResponse.apply(ParseObjectListFromResponse.java:43)
        at org.jclouds.rest.internal.InvokeHttpMethod.invoke(InvokeHttpMethod.java:90)
        at org.jclouds.rest.internal.InvokeHttpMethod.apply(InvokeHttpMethod.java:73)
        at org.jclouds.rest.internal.InvokeHttpMethod.apply(InvokeHttpMethod.java:44)
        at org.jclouds.reflect.FunctionalReflection$FunctionalInvocationHandler.handleInvocation(FunctionalReflection.java:117)
        at com.google.common.reflect.AbstractInvocationHandler.invoke(AbstractInvocationHandler.java:87)
        at com.sun.proxy.$Proxy75.list(Unknown Source)
        at org.jclouds.openstack.swift.v1.blobstore.RegionScopedSwiftBlobStore.list(RegionScopedSwiftBlobStore.java:200)
```

**Comment by timuralp** [comment]
I can take a look at that. Should be a straightforward fix.

**Comment by kahing** [comment]
addressed by https://github.com/bouncestorage/swiftproxy/pull/27

**Comment by kahing** [comment]
merged #27
